packages/e2e-test-app-fabric/windows/ExperimentalFeatures.props

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Label="Microsoft.ReactNative Experimental Features">
+    <UseHermes>true</UseHermes>
+    <UseFabric>true</UseFabric>
+    <UseWinUI3>false</UseWinUI3>
+
+    <ReactExperimentalFeaturesSet>true</ReactExperimentalFeaturesSet>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Unpackaged RNTesterApp-Fabric" Condition="'$(SolutionName)'=='RNTesterApp-Fabric'">
+    <!--
+      WARNING - DO NOT MAKE THE VERSION OF MICROSOFT.UI.XAML BE ANYTHING BUT PRERELEASE.
+      Playground-Win32 is an unpackaged win32 app, and needs to use Microsoft.UI.Xaml from a prerelease,
+      to be able to carry WinUI in-app instead of using the Framework Package
+    -->
+    <WinUI2xVersion>2.7.0-prerelease.210913003</WinUI2xVersion>
```

Comment:
Right now since no hosting of XAML is being done, it should all work with Win2 or 3.  But when we get to hosting that would be 3 only.  So we should probably make the test app winui3 from the start.

Alternatively, we could make the test app a win32 app with no XAML.  -- Which I think aligns with the playground app.

Reply:
As this is going to inform our app template, I would love to leave XAML out for now if possible. It would be nice to make XAML strictly opt-in (esp. if we end up not needing XAML for any core components).